jekyll/_cci2/runner-installation.adoc

```diff
@@ -194,6 +196,34 @@ mkdir -p /opt/circleci/workdir
 chown -R circleci /opt/circleci/workdir
 ```
 
+=== Configure SELinux policy (RHEL 8)
+
+An SELinux policy is required for runner to accept and launch jobs on RHEL 8 systems. Note that this policy does not add any permissions to the ones that may be required by individual jobs on this runner install.
```

Comment:
It may be worth noting that this policy is not supported on earlier versions of RHEL (<= 7)